homework5-2.py

```diff
@@ -0,0 +1,10 @@
+text = str(input('Enter words: '))
+lst = text.replace('\n', '').split()
+dct = {}
+for element in lst:
+    dct[element] = dct.get(element, 0) + 1
+lst1 = sorted(dct.values())
+value_max = lst1[-1]
```

Comment:
please separate logical blocks with empty lines. 
overall, this is the best solution on this moment form all group